src/lib/cursorBasedPagination.ts

```diff
@@ -31,7 +31,9 @@ export function cursorBasedPagination<T extends CursorBasedPagination>(
           ...pageInfo,
           // reduce total count by excluding dangling items so it won't cause a new page query
           // after item was removed from the cache (for .e.g deleted publication)
-          totalCount: pageInfo?.totalCount - danglingItems?.length
+          totalCount: pageInfo?.totalCount
```

Comment:
@bigint after running a `yarn graphql-codegen` totalCount became nullable which was causing a [type check issue](https://github.com/lensterxyz/lenster/actions/runs/3234597297/jobs/5298074546#step:7:8). Want to make sure you see this 👀

Reply:
Should be fixed now 😊 can you revert and update the branch 🙇🏼

Reply:
Appreciate you! Done!